examples/research_projects/jax-projects/README.md

```diff
@@ -0,0 +1,149 @@
+
+## Important dates
+
+**23.06.** Official announcement of the community week. Make sure to sign-up in [this google form](https://forms.gle/tVGPhjKXyEsSgUcs8).
+**23.06. - 30.06.** Participants will be added to an internal Slack channel. Project ideas can be proposed here and groups of 3-5 are formed. Read this document for more information. 
+**30.06.** Release of all relevant training scripts in JAX/Flax as well as other documents on how to set up a TPU, how to use the training scripts, how to submit a demo, tips & tricks for JAX/Flax, tips & tricks for efficient use of the   hub. 
+**30.06. - 2.07.** Talks about JAX/Flax, TPU, Transformers, Computer Vision & NLP will be held. 
+**7.07.** Start of the community week! Access to TPUv3-8 will be given to each team.
+**7.07. - 14.07.** The Hugging Face & JAX/Flax & Cloud team will be available for any questions, problems the teams might run into.
+**15.07.** Access to TPU is deactivated and community week officially ends.
+**16.07.** Deadline for each team to submit a demo. 
+
+## Communication
+
+All important communication will take place in an internal Slack channel, called `#flax-jax-community-week`.
+Important announcements of the Hugging Face, Flax/JAX, and Google Cloud team will be posted there. Such announcements include general information about the community week (Dates, Rules, ...), release of relevant training scripts (Flax/JAX example scripts for NLP and Vision), release of other important documents (How to access the TPU), ... 
+The Slack channel will also be the central place for participants to post about their results, share their learning experiences, ask questions, etc...
+
+All important communication will take place in an internal Slack channel, called `#flax-jax-community-week`. 
+Important announcements of the Hugging Face, Flax/JAX, and Google Cloud team will be posted there. 
+Such announcements include general information about the community week (Dates, Rules, ...), release of relevant training scripts (Flax/JAX example scripts for NLP and Vision), release of other important documents (How to access the TPU), etc. 
+The Slack channel will also be the central place for participants to post about their results, share their learning experiences, ask questions, etc.
+
+To be invited to the Slack channel, please make sure you have signed up [on the Google form](https://forms.gle/tVGPhjKXyEsSgUcs8). 
+
+**Note**: If you have signed up on the google form, but you are not in the Slack channel, please leave a message on [(TODO) the official forum announcement]( ) and ping `@Suzana` and `@patrickvonplaten`.
+
+## Projects
+
+During the first week after the community week announcement, **23.06. - 30.06.**, teams will be formed around the most promising and interesting project ideas. Each team can consist of 2 to 5 participants. Projects can be accessed [here](https://discuss.huggingface.co/c/flax-jax-projects/22).
```

Comment:
2 to 5 or 3 to 5 participants?

Reply:
thought 2 can be enough as well...think it doesn't hurt to stay flexible here